src/Auth/GuardDirective.php

```diff
@@ -92,6 +95,7 @@ protected function authenticate(array $guards): void
      * Handle an unauthenticated user.
      *
      * @param  array<string|null>  $guards
+     * @throws AuthenticationException
```

Comment:
```suggestion
     * @return never
```

Should also make the phpstan config change obsolete

Reply:
I kept both